virttest/shared/deps/run_autotest/kernel_install/kernelinstall.py

```diff
@@ -83,7 +83,7 @@ def _kernel_install_src(self, base_tree, config=None, config_list=None,
             base_tree = os.path.join(self.bindir, base_tree)
         if not utils.is_url(config):
             config = os.path.join(self.bindir, config)
-        kernel = self.job.kernel(base_tree, self.outputdir)
+        kernel = self.job.kernel(base_tree, self.logdir)
```

Comment:
Hi @pevogam, the problem with `outputdir` is fixed by #3214. So I think this change is not necessary.

Reply:
Thanks for talking a look, this is just some fixes I collected that were lacking on master. Alright, I will take a look into your PR first and then see what is left here.